packages/jats-xml/src/validate/dtd.ts

```diff
@@ -245,9 +245,8 @@ async function ensureDtdExists(session: ISession, opts: JatsOptions) {
     await ensureDtdZipExists(session, opts);
     const zipFile = localDtdZipFile(opts);
     session.log.info(`🤐 Unzipping template: ${zipFile}`);
-    await createReadStream(zipFile)
-      .pipe(unzipper.Extract({ path: opts.directory }))
-      .promise();
+    const zip = new AdmZip(zipFile);
+    zip.extractAllTo(opts.directory);
```

Comment:
Is it syncronous?

Reply:
Yes! I think probably ok - nothing parallel is happening here.